Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -396,6 +400,7 @@ public GiveSkill(ContentXElement element, string parentDebugName)
                 TriggerTalents = element.GetAttributeBool(nameof(TriggerTalents), true);
                 UseDeltaTime = element.GetAttributeBool(nameof(UseDeltaTime), false);
                 Proportional = element.GetAttributeBool(nameof(Proportional), false);
+                ProportionalMaxAmount = element.GetAttributeFloat(nameof(ProportionalMaxAmount), 2);
```

Comment:
This change will cause backwards compatibility problems, because the value defaults to 2 (when previously there was no limit). Items that used to give more than 2 proportional skill will now only give up to 2.